pkg/types/kustomization.go

```diff
@@ -71,10 +72,10 @@ type Kustomization struct {
 	// and http://jsonpatch.com
 	PatchesJson6902 []patch.Json6902 `json:"patchesJson6902,omitempty" yaml:"patchesJson6902,omitempty"`
 
-	// ImageTags is a list of (image name, new tag) pairs for simply
-	// changing a an image tag.  This can also be achieved with a
+	// Image is a list of (image name, new name, new tag or digest)
+	// for changing image names, tags or digests. This can also be achieved with a
 	// patch, but this operator is simpler to specify.
-	ImageTags []ImageTag `json:"imageTags,omitempty" yaml:"imageTags,omitempty"`
+	Image []image.Image `json:"images,omitempty" yaml:"images,omitempty"`
```

Comment:
Change it to `Images` since it is an array of `Image`.

Reply:
👍